draft-smyslov-ipsecme-ikev2-downgrade-prevention.xml

```diff
@@ -38,8 +39,13 @@
             <t> The Internet Key Exchange version 2 protocol (IKEv2) defined in <xref target="RFC7296" /> provides
             authenticated key exchange in the IP Security (IPsec) architecture. The cryptographic design of IKEv2 is based
             on SIGMA protocol defined in <xref target="SIGMA" />. The protocol allows peers to mutually
-            authenticate themselves and to derive session keys, that are used to protect traffic.
+            authenticate themselves and to derive session keys that are used to protect traffic.
             </t>
+
+            <t>(RFC EDITOR: Please remove this paragraph.) This document is being developed at
+            https://github.com/smyslov/ikev2-downgrade-prevention.

```

Comment:
What XML syntax do we use to make this a hyperlink?

Reply:
<eref target="https://github.com/smyslov/ikev2-downgrade-prevention"/>